### DISCUSSION TITLE
No changelog for pypi release

### DISCUSSION THREAD

**Discussion by dtrckd** [confident-ai/deepeval#1238] [Q&A]
Hi,

There is new releases on pypi and the deepeval lib invite the user to upgrade at runtime with text such as : 
```
UserWarning: You are using deepeval version 2.0.5, however version 2.0.9 is available.
You should consider upgrading via the "pip install --upgrade deepeval" command.
```

But, I am unable to see changelogs on the repo or in the GH release  page that corresponds to those versions.
Is there a way to understand to what changes corresponds those versions ?

**Comment by dtrckd** [comment]
The pip package version does not match any tags or release version present the github repo.
If someone is able to understand what changes is associated to a given pip version here, I would be delighted to share his knowledge.

**Comment by penguine-ip** [comment]
hey @dtrckd we'll try to update the changelogs once a week in the future
